preview/html/testpage.pug

```diff
@@ -3,7 +3,7 @@ html
 	head
 		meta(charset='utf-8')
 
-		link(href='font/font.css', rel='stylesheet', type='text/css')
+		link(href='font/SimpleIcons.css', rel='stylesheet', type='text/css')
```

Comment:
I would prefer the filename to be all lowercase (either _"simpleicons.css"_ or _"simple-icons.css"_) Could you update that?